designs/auto-certs-renewal.md

```diff
@@ -0,0 +1,126 @@
+# Auto Certificates Renewal
+
+## Context
+
+As a user who deployed K8s cluster with EKS-A, I want an option to automatically renew the certificates in etcd and control plane nodes before expiry in 1 year.
```

Comment:
Hey @d8660091, the problem this is seeking to solve is a bit unclear to me. I'd like to see these questions answered.

- What problems are you solving?
- How do they impact users?